ang/civicase-base/filters/crm-date.filter.js

```diff
@@ -0,0 +1,29 @@
+(function ($, angular) {
+  var module = angular.module('civicase-base');
+
+  module.filter('crmDate', crmDate);
```

Comment:
As this is defined in Civicase, we should name if `civicaseCrmDate` or just `civicaseDate`.